app/src/main/java/com/google/samples/apps/topeka/widget/quiz/SelectItemQuizView.java

```diff
@@ -59,7 +58,8 @@ protected View createQuizContentView() {
             @Override
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 allowAnswer();
-                toggleAnswerFor(position);
+                resetAnswer();
+                getAnswers()[position] = true;
```

Comment:
I would change it to variable `mAnswers`. Function `getAnswers` should be used only once in class constructor to generate new or get existing user choices.